### PR TITLE
Add helm chart

### PR DESCRIPTION
**Pull request by coffeefreak101**
- Moved the helm chart from sandbox to here.
- Updated golangci version
- Fixed some logging issues I noticed

### CODE REVIEW

File: chart/templates/bioscfg-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  name: bioscfg-config
+  namespace: default
+data:
+  config.yaml: |
+    log_level: debug
+    concurrency: 5
+    nats:
+      app_name: bioscfg
+      # TODO: remove deprecated stream_urn_ns field
+      stream_urn_ns: hollow-controllers
```
Comment (DoctorVin) [inline chart/templates/bioscfg-configmap.yaml:14]: copypasta? you shouldn't need `stream_urn_ns`

**Review by DoctorVin — APPROVED** [review]
looks fine to me. I noticed some old copy-pasta you might want to excise.

File: chart/templates/bioscfg-configmap.yaml (hunk 1 above)
Comment (coffeefreak101) [reply chart/templates/bioscfg-configmap.yaml]: Awesome, thanks for the info!

**Review by DoctorVin — APPROVED** [review]
Thank you!

**Pull request merged**